src/components/AutoCompleteSuggestions/BaseAutoCompleteSuggestions.tsx

```diff
@@ -47,6 +47,7 @@ function BaseAutoCompleteSuggestions<TSuggestion>(
     const StyleUtils = useStyleUtils();
     const rowHeight = useSharedValue(0);
     const scrollRef = useRef<FlashList<TSuggestion>>(null);
+
```

Comment:
This line break is irrelevant to this PR

Reply:
removing rn

Reply:
done